engine/apps/email/tests/test_inbound_email.py

```diff
@@ -0,0 +1,19 @@
+import pytest, anymail, json
+from apps.email.inbound import InboundEmailWebhookView
+from django.urls import reverse
+from rest_framework.test import APIClient
+
+@pytest.mark.django_db
+def test_amazon_ses_provider_load(
+    settings
+):
+    settings.INBOUND_EMAIL_ESP = "amazon_ses"
+
+    test_data = {
+        "foo" : "bar"
+    }
+    inbound_view = InboundEmailWebhookView()
+    client = APIClient()
+    url = reverse("integrations:inbound_email_webhook")
+    with pytest.raises(anymail.exceptions.AnymailAPIError):
+        response = client.post(url, data=test_data)
```

Comment:
👋 what's the thought behind asserting that `anymail.exceptions.AnymailAPIError` is raised? I would 

What about asserting that you get back HTTP 200 from this request? You could even go one step further and mock and assert that `create_alert` is called as expected for the payload you pass in ([handler code](https://github.com/grafana/oncall/blob/dev/engine/apps/email/inbound.py#L64-L79)).

Reply:
Asserting 200/OK looks more complicated to test, but is still possible